extension/src/popup/basics/buttons/Button/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Button as SDSButton } from "@stellar/design-system";
+
+import "./styles.scss";
+
+// override SDS Button css with freighter design
+
+enum ButtonVariant {
+  primary = SDSButton.variant.primary,
+  tertiary = SDSButton.variant.tertiary,
+}
+
+interface ButtonComponent {
+  variant: typeof ButtonVariant;
+}
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  isLoading?: boolean;
+  fullWidth?: boolean;
+  variant?: ButtonVariant;
+}
+
+export const Button: React.FC<ButtonProps> & ButtonComponent = ({
+  children,
+  isLoading,
+  fullWidth,
+  variant = ButtonVariant.primary,
+  ...props
+}: ButtonProps) => {
+  const sdsVariant =
+    variant === ButtonVariant.primary
+      ? SDSButton.variant.primary
+      : SDSButton.variant.tertiary;
+  const sds = SDSButton({
+    variant: sdsVariant,
+    fullWidth,
+    children,
+  });
+
+  return (
+    <SDSButton
+      className={`BasicButton BasicButton--${variant} ${sds?.props?.className}`}
```

Comment:
kind of weird, but didn't want to wrap the SDSButton in a div that could mess up styling (eg. like [here](https://github.com/stellar/freighter/blob/126ef9dd2ca89111a7c0f180d3ecbf3b98eb3501/extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss#L69)). So creating the SDS button and then adding styling to it instead. This way if we move back to SDS we can just change the imports again

Reply:
This is great! I like this is strategy.

Let's definitely work under the assumption that these will be in SDS sooner rather than later